apps/explorer/src/ui/DateCard.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Text } from '~/ui/Text';
+import { formatDate } from '~/utils/timeUtils';
+
+export interface DateCardProps {
+    date: Date | number;
+}
+
+// TODO - add format options
+export function DateCard({ date }: DateCardProps) {
```

Comment:
As it exists here I'm not convinced this is really justifies being a separate component. It seems like it's just one text node (+ colors, which we can fix later). I think we could justify it a little more if we rendered it in a time tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

Reply:
The idea here is to expand this component to support different styles and formats depending on the use case.

Reply:
Sure, I still think this should be in a time tag?